wordpress-mobile/WordPress-Android
# [Site Creation] Site Preview Landscape Layout

Fixes #8687. This PR adds an alternate layout for the landscape orientation of the site preview page.

Here are screenshots for the portrait and landscape orientations:

![screenshot_1548726074](https://user-images.githubusercontent.com/662023/51878266-11caee80-233d-11e9-8b22-87e1c37995ee.png)
![screenshot_1548726053](https://user-images.githubusercontent.com/662023/51878247-04adff80-233d-11e9-859f-d43fc1c388e3.png)

I've recorded gifs to showcase the animations. I've slowed the animation down to make the review easier. [Portrait](https://cloudup.com/cwLb86u4lUv), [Landscape](https://cloudup.com/cILlzs78ADZ)

* In these gifs a site is not actually created, I am loading one of my test sites in it. (for faster testing)
* The last few frames in the portrait animation seems to be cut off, or playing too fast. Please ignore that, the site loads just fine.
* The landscape one was rotated after its recorded. That's why it's lower quality.

**To test:**
* Enable the `NEW_SITE_CREATION_ENABLED` in `WordPress/build.gradle`.
* Go through the site creation flow in both portrait and landscape modes and verify that everything looks good and matches the screenshots above.

Update release notes:
